docs/csharp/tutorials/pattern-matching.md

```diff
@@ -193,7 +193,7 @@ vehicle switch
 };
 ```
 
-The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra $5.00. Light trucks under 3000 lbs are given a $2.00 discount. That rule is implemented with the following code:
+The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra &#36;5.00. Light trucks under 3000 lbs are given a &#36;2.00 discount. That rule is implemented with the following code:
```

Comment:
```suggestion
The toll authority isn't concerned with the number of passengers in the delivery trucks. Instead, they charge more based on the weight class of the trucks. Trucks over 5000 lbs are charged an extra %245.00. Light trucks under 3000 lbs are given a %242.00 discount. That rule is implemented with the following code:
```

Reply:
@rpetrusha thanks! Let's see; if that one fails, I'll rephrase the sentence.

Reply:
One other suggestion: We could split those paragraphs into multiple lines, so the `$` is on multiple source lines.

Reply:
@BillWagner thanks! it's simpler indeed. Would be the next option then. Next-next is to format as a list.